### DISCUSSION TITLE
Game Element Ideas

### DISCUSSION THREAD

**Discussion by anythingapplied** [jonathan-d-zhang/flawless#11] [Ideas]
Ideas for game elements.  We'll need to order these at some point so that we can introduce them one at a time.

* Pickups:
    * Keys - Opens any regular door
    * Escape Key - Only key that opens the exit
    * Shiv - Allows you to kill a cop by stepping into the same gridspace as them
* Fixtures
    * Door - Requires one key. Stays open after use.
    * Open Door
    * Level Exit Door
    * Vent - Cannot be seen if standing on this gridspace. Guards can walk over you.
    * Arrow - Will turn a guard who steps on it
    * Lasers/Cameras - Fixtures that can spot you and move in predictable ways
* Units
    * Guards - Turn around when they hit a wall.  Follows arrows that it hits. Can see x spaces directly in front of them
    * Lookout - Rotates through the 4 cardinal directions, seeing x spaces directly in front of them (maybe this should just be the camera?)
* Moveables
    * Blocks - Pushable objects.  Guards can't push them and will turn around if they run into them.

**Comment by GDWR** [comment]
Line of sight rather than cone of sight for guard vision as discussed. Can lead to better level design
